miqa/core/migrations/0023_user_identified_artifacts.py

```diff
@@ -14,8 +12,6 @@ class Migration(migrations.Migration):
         migrations.AddField(
             model_name='scandecision',
             name='user_identified_artifacts',
-            field=models.JSONField(
-                default=miqa.core.models.scan_decision.default_identified_artifacts
-            ),
+            field=models.JSONField(default=dict),
```

Comment:
In general, it's bad practice to edit a django migration file manually. For example, when I ran migrate after these changes, this change was not applied because django does not look back in the old migration files if it already has record of applying them. I'm going to undo these changes and include them in 0036.